manifests/m/Mozilla/sccache/0.3.1/Mozilla.sccache.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=MDSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Mozilla.sccache
+PackageVersion: 0.3.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: zip
+InstallModes:
+- silent
+UpgradeBehavior: uninstallPrevious
+Commands:
+- sccache
+Installers:
+- Architecture: x64
+  InstallerType: zip
```

Comment:
Zip installers must have the nested installer type and nested installer files set

Reply:
The 1.2.0 JSON schema doesn't support those keys though. 1.4.0 support I couldn't get working locally, though it does support those keys. I have an orphaned commit with my attempt at doing that, I'll comment with the commit hash when I'm back at my computer, so maybe you can tell me what I'm missing.

Reply:
> The 1.2.0 JSON schema doesn't support those keys though. 1.4.0 support I couldn't get working locally, though it does support those keys. I have an orphaned commit with my attempt at doing that, I'll comment with the commit hash when I'm back at my computer (so maybe you can tell me what I'm missing).

1.4 is officially supported; I suggest using `YamlCreate.ps1 Mozilla.sccache 0.3.1 2` which will re-prompt for the installer URL, walk you through the nested files, and update all the manifests to v1.4